src/domain/entities/noteSettingsPublic.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Notes settings entity
+ */
+export default interface NoteSettingsPublic {
```

Comment:
Do we need this? Client probably does not need "noteId" inside NoteSettigns since it probably already knows it while requesting.

Reply:
It is mainly needed for the case, where the user is creator and the note is not public
`
 test('Returns note by public id with 200 status when access is disabled, but user is creator', async () => {
      const expectedStatus = 200;
      const userId = 1;
      const accessToken = global.auth(userId);
      const expectedResponse = {
        'note': {
          'id': '73NdxFZ4k7',
          'creatorId': 1,
          'content': null,
          'createdAt': '2023-10-16T13:49:19.000Z',
          'updatedAt': '2023-10-16T13:49:19.000Z',
        },
        'noteSettings':  {
          'customHostname': 'codex.so',
          'isPublic': false,
        },
        'accessRights': {
          'canEdit': true,
        },

      };
`

Reply:
It is mainly needed for the case, where the user is creator and the note is not public, to know the isPublic


```
test('Returns note by public id with 200 status when access is disabled, but user is creator', async () => {
      const expectedStatus = 200;
      const userId = 1;
      const accessToken = global.auth(userId);
      const expectedResponse = {
        'note': {
          'id': '73NdxFZ4k7',
          'creatorId': 1,
          'content': null,
          'createdAt': '2023-10-16T13:49:19.000Z',
          'updatedAt': '2023-10-16T13:49:19.000Z',
        },
        'noteSettings':  {
          'customHostname': 'codex.so',
          'isPublic': false,
        },
        'accessRights': {
          'canEdit': true,
        },

      };

Reply:
Still not clear for me. Wy do we need to know isPublic in that case? Get Note either returns a Note or returns an exception.